icu4j/build.properties

```diff
@@ -8,5 +8,5 @@ api.report.version = 70
 api.report.prev.version = 69
 release.file.ver = 70rc
 api.doc.version = 70 Release Candidate
```

Comment:
This is used for API doc. We agreed to generate release API doc at RC, so we want to change this to '70.1' instead of '70 Release Candidate'.

Reply:
Should we add this to the documentation too? Right now it says:
> api.doc.version: The version displayed in API reference doc - use full version number such as “60.1” for official releases

Can we change it to something like
> api.doc.version: The version displayed in API reference doc - use full version number such as “60.1” for official and RC releases

Reply:
Yes. We discussed about this in last release. And BRS document should be updated. It's extremely rare that we need to update API doc between RC and GA, and we should not require publishing them twice with same content.

Reply:
Thanks Yoshito! @erik0686 Can you update the documentation in `versions.md` as part of this change as well?

Reply:
Yep!